src/tests/CoinDetailsPage.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
```

Comment:
- [ ] 🙏Please address the failing❌ test in this file. By doing so, you will enhance the functionality and reliability of your project, as the code has already been thoroughly tested.

Reply:
All tests passed on my end. Could you explain which tests you are talking about?  @abdallahmalima 

Please make sure to install all dependencies as I am also using react-test-renderer.

![image](https://github.com/bilalrajput09/CryptoWatch-React-App/assets/116716918/8bb81982-b250-43ea-b20e-0d88cfaea150)